includes/class-wc-payments-features.php

```diff
@@ -19,7 +19,7 @@ class WC_Payments_Features {
 	 * @return bool
 	 */
 	public static function is_grouped_settings_enabled() {
-		return '1' === get_option( '_wcpay_feature_grouped_settings', '0' );
+		return '1' === get_option( '_wcpay_feature_grouped_settings', '1' );
```

Comment:
Per discussion with @waclawjacek and @frosso , we are keeping this feature flag but defaulting it to `1`. This technique allows us to display the new layout without adding a new entry into the DB option table. Worst case, we can roll out patches to disable the feature by manipulating `_wcpay_feature_grouped_settings`.